### DISCUSSION TITLE
Remember keyboard brightness with "performance mode"

### DISCUSSION THREAD

**Discussion by TheBestPessimist** [seerge/g-helper#1933] [Ideas]
I would like when i'm on `Silent` mode, that my keyboard backlight is changed to low brightness (level 1/3), and when i am on `Balanced` or `Turbo` to have it at max brightness.

I am thinking that what i want can can be made even smarter by remembering the brightness per each mode. This makes it such that on battery, the keyboard brightness is automatically turned down too.

WDYT? Is this something that you consider adding? iI yes, i could try to make the PR myself.

**Comment by seerge** [comment]
@TheBestPessimist hmm, i think saving backlight brightness per performance mode can be confusing :) and people will just start reporting bugs that their brightness changes out of the blue.

What is the practical sense ? App already remembers 2 brightness level : on battery and when plugged. And this is logical, cause it's related to battery life and usage scenarios :|
